UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/PlayFabMultiplayerAgentAPI.cs

```diff
@@ -280,7 +280,13 @@ public static IEnumerator SendHeartBeatRequest()
          
            
         }
-        private static void ProcessAgentResponse(HeartbeatResponse heartBeat)
+
+#if UNIT_TESTING
+        public
```

Comment:
curious, internal wouldn't work?

Reply:
No, the tests are in a separate assembly from the SDK

Reply:
What do you mean “assembly”? In the Unity world, folks just re-use the files in the assets folder, no?